### DISCUSSION TITLE
Learning resources & suggestions

### DISCUSSION THREAD

**Discussion by LukyGuyLucky** [c3lang/c3c#1134] [Q&A]
I like C3 is so much close to C  and would like to learn more.Moreover,I have a rough idea that I would like to share my learning stuffs step by step in local community to let more friends know C3 and ideally learning this beauitful language together. So may I know
1. whether the resources in C3/test folder are good to learn starting with?
2.  is there any other resources do help?
3.  whether translating some c code is a good idea?

Thanks in advance.

Regards.

**Comment by data-man** [comment]
There is https://learn-c3.org.
It's the unofficial resource if I don't remember correctly. 😀

**Reply by LukyGuyLucky** [reply]
Thanks for the feedback.
Yes,I am already there.It is very good,it could be great if there are more topics regarding to generics,compile time,macro sytem.etc.

**Reply by lerno** [reply]
There is a new website in development, but in the mean time you'll have to use c3-lang.org or ask on the Discord or here.

**Reply by LukyGuyLucky** [reply]
> There is a new website in development


Very excited to hear.

Discord is not an option to me for some reason.Sorry for bothering the Dev. Team here.

**Reply by lerno** [reply]
There is also: https://c3lang.discourse.group

**Reply by LukyGuyLucky** [reply]
Got it.I will be there.
Thanks.

**Comment by lerno** [comment]
Better look in /resources, it contains various samples. As @data-man said, there is also learn-c3.org. As for converting C code to learn: yes, you could certainly do that.

**Reply by LukyGuyLucky** [reply]
Well noted with thanks.

BTW,may I ask how to build resources/nolibc example under windows 10 64 bit?I have vs2019 community,msys2 (gcc 13) installed,no llvm.

**Reply by lerno** [reply]
It only works on Linux as it uses syscalls.

**Reply by LukyGuyLucky** [reply]
Thanks again.
